core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -103,6 +103,23 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
       }.recover(bigQueryRecover)
     }
 
+  def deleteEntityAttribute(workspaceName: WorkspaceName, entityType: String, entityAttributeNamespace: String, entityAttributeName: String, dataReference: Option[DataReferenceName], billingProject: Option[GoogleProjectId]): Future[PerRequestMessage] =
+    getWorkspaceContextAndPermissions(workspaceName, SamWorkspaceActions.write, Some(WorkspaceAttributeSpecs(all = false))) flatMap { workspaceContext =>
+      val entityRequestArguments = EntityRequestArguments(workspaceContext, userInfo, dataReference, billingProject)
+
+      val deleteFuture = for {
+        entityProvider <- entityManager.resolveProviderFuture(entityRequestArguments)
+        _ <- entityProvider.deleteEntityAttribute(workspaceContext, entityType, entityAttributeNamespace, entityAttributeName)
+      } yield {
+        PerRequest.RequestComplete(StatusCodes.NoContent)
+      }
+
+      deleteFuture.recover {
+        case delEx: DeleteEntitiesConflictException => RequestComplete(StatusCodes.Conflict, delEx.referringEntities)
+      }.recover(bigQueryRecover)
```

Comment:
I don't think the `bigQueryRecover` is helpful here - that's meant to trap BigQuery-related exceptions

Reply:
I was trying to write this method similarly to `deleteEntities` above it so that we could eventually add a deleteAttribute feature for data repo snapshot tables too. But that might be overkill right now, so I could just simplify it.